### ISSUE TITLE
Add network ids to /protocols endpoint

### ISSUE THREAD

**Issue by tboeckmann** [mailchain/mailchain#281]
**Is your feature request related to a problem? Please describe.**
In order to support substrate, the /protocols endpoint should return network ID with the network name in the response for each protocol.

**Describe the solution you'd like**
Upgrade the existing response to include the `name` & `id` fields as strings. Empty `id` values should be returned as empty strings.

E.g.:
```
[
  {
    "name": "string",
    "networks": [
      // "string"    <-- previous value
      { "name": "string", "id": "string" }
    ]
  }
]
```

**Describe alternatives you've considered**

**Additional context**

**Comment by tboeckmann** [comment]
See here for reference to Network IDs in substrate:
https://github.com/paritytech/substrate/commit/cf98501f92a98ab7133829a746128be98aea0117#diff-a40465c5924214464e991fe6c9deb541R327
&
https://github.com/paritytech/substrate/pull/3147

**Comment by robdefeo** [comment]
closed by #401

**Comment by robdefeo** [comment]
closed by #401
